src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -522,11 +522,6 @@ static void ConfigureFeatureFlags()
 #if __SKIA__
 			Uno.UI.FeatureConfiguration.ToolTip.UseToolTips = true;
 #endif
-
-#if HAS_UNO
-			// Allow template pool to work under higher memory load for CI.
-			FrameworkTemplatePool.HighMemoryThreshold = 0.9f;
-#endif
```

Comment:
Why did you remove this one?

Reply:
@jeromelaban It was added as a way to allow the pooling to work in CI when asked. The `MockProvider` I added achieves the exact same, but in a more reliable way.

The `0.9f` worked reliably for CI, but it doesn't always work locally.

Reply:
For reference, https://github.com/unoplatform/uno/pull/10725